.github/workflows/test.yml

```
@@ -20,201 +16,6 @@ permissions:
-        name: Restore cached fetch params outputs
-        uses: actions/cache/restore@v4
-        with:
-          key: ${{ steps.cache.outputs.fetch_params_key }}
-          path: ${{ steps.cache.outputs.fetch_params_path }}
-          fail-on-cache-miss: true
-      # TODO: Install statediff (used to be used for conformance)
-      - name: Create temporary directory for reports
-        id: reports
-        run: mktemp -d | xargs -0 -I{} echo "path={}" | tee -a $GITHUB_OUTPUT
-      # TODO: Track coverage (used to be tracked for conformance)
-      - name: Run tests
-        env:
-          NAME: ${{ matrix.name }}
-          LOTUS_SRC_DIR: ${{ github.workspace }}
-          REPORTS_PATH: ${{ steps.reports.outputs.path }}
-          SKIP_CONFORMANCE: ${{ fromJson(steps.group.outputs.metadata).skip_conformance && '1' || '0' }}
-          TEST_RUSTPROOFS_LOGS: ${{ fromJson(steps.group.outputs.metadata).test_rustproofs_logs && '1' || '0' }}
-          FORMAT: ${{ fromJson(steps.group.outputs.metadata).format || 'standard-verbose' }}
-          PACKAGES: ${{ join(fromJson(steps.group.outputs.metadata).packages, ' ') }}
-        run: |
-          gotestsum \
-            --format "$FORMAT" \
-            --junitfile "$REPORTS_PATH/$NAME.xml" \
-            --jsonfile "$REPORTS_PATH/$NAME.json" \
-            --packages="$PACKAGES" \
-            -- ${{ fromJson(steps.group.outputs.metadata).go_test_flags || '' }}
-      - name: Modify junit.xml for BuildPulse
-        env:
-          NAME: ${{ matrix.name }}
-          REPORTS_PATH: ${{ steps.reports.outputs.path }}
-          PACKAGES: ${{ join(fromJson(steps.group.outputs.metadata).packages, ' ') }}
-        if: always()
-        run: |
-          # Modify test suite name and classname attributes in JUnit XML for better grouping
-          # in BuildPulse. itests are run with go test ./itests/file_test.go and therefore Go
-          # assigns the name and classname attributes to "command-line-arguments". Others get the
-          # package name for both.
-          if [[ "${{ matrix.name }}" == itest-* ]]; then
-            PACKAGE_NAME=$(basename "$PACKAGES" .go)
-            sed -i 's/ name="command-line-arguments"/ name="itests"/g' "$REPORTS_PATH/$NAME.xml"
-            sed -i 's/classname="command-line-arguments"/classname="'"$PACKAGE_NAME"'"/g' "$REPORTS_PATH/$NAME.xml"
-          else
-            sed -i 's# name="github.com/filecoin-project/lotus/\(.*\)"# name="'${{ matrix.name }}':\1"#g' "$REPORTS_PATH/$NAME.xml"
-          fi
-          cat "$REPORTS_PATH/$NAME.xml"
-      - if: success() || failure()
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ matrix.name }}-${{ runner.os }}-${{ runner.arch }}
-          path: |
-            ${{ steps.reports.outputs.path }}/${{ matrix.name }}.xml
-            ${{ steps.reports.outputs.path }}/${{ matrix.name }}.json
-        continue-on-error: true
+    name: Test
+    uses: ./.github/workflows/reusable-test.yml
```

```suggestion
    uses: ./.github/workflows/reusable-test.yml
    with:
      run_very_expensive_tests: false
```

I know that's the default, but it makes it explicit that this workflow isn't running with expensive tests.